src/os/debug/fancierwindow.js

```diff
@@ -62,6 +63,14 @@ os.debug.FancierWindow.prototype.onBeforeUnload_ = function() {
 };
 
 
+/**
+ * @public
+ */
+os.debug.FancierWindow.prototype.closeLogger = function() {
+  // this.exit_ is private (grr), so copy it here
+  this.onBeforeUnload_();
+};
+
```

Comment:
`onBeforeUnload_` is defined in `os.debug.FancierWindow` itself. Could just rename it to `closeLogger` and make it public instead of wrapping it if it is all the same. Also, is that comment copy pasta?

Reply:
I agree with renaming `onBeforeUnload_ -> closeLogger`.

As a cleanup step, we could call `this.exit_(null)` (it doesn't use the event anyway) and `@suppress {accessControls}` instead of copying the function contents. This may have been written before we started making use of `@suppress`.

Reply:
Yeah, I that comment was copied and pasted, I think I was doing something else and then forgot to delete it. I can get rid of it though